### DISCUSSION TITLE
GUI application for PyMuPDF

### DISCUSSION THREAD

**Discussion by Augus1999** [pymupdf/PyMuPDF#2113] [Show and tell]
Hello all,
I've written a GUI application using PyMuPDF and PyQt5: [pyPDFeditor-GUI](https://github.com/Augus1999/pyPDFeditor-GUI). I hope this will be a useful tool. 🥰

**Comment by JorjMcKie** [comment]
Interesting stuff, and thank you for announcing it here!
I hope many will give it a try.
